### PR TITLE
Fix setDeprecated deprecation

### PR DESCRIPTION
**Pull request by dannyvw**
| Q               | A
| --------------- | -----
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | yes 
| Related tickets | partially #335 
| License         | MIT

### CODE REVIEW

**Review by vvasiloi — APPROVED** [review]

File: src/Bundle/DependencyInjection/Compiler/PagerfantaBridgePass.php [hunk 1]
```diff
@@ -48,12 +49,23 @@ private function aliasRenamedServices(ContainerBuilder $container): void
...
-                ->setDeprecated(true, 'The "%alias_id%" service alias is deprecated since Sylius 1.8, use the "pagerfanta.view_factory" service ID instead.');
+                ->setDeprecated(...$this->getDeprecationMessage());
         }
     }
+
+    private function getDeprecationMessage(): array
+    {
+        $message = 'The "%alias_id%" service alias is deprecated since Sylius 1.8, use the "pagerfanta.view_factory" service ID instead.';
+
+        if (method_exists(Alias::class, 'getDeprecation')) {
```
Comment (lchrusciel) [inline src/Bundle/DependencyInjection/Compiler/PagerfantaBridgePass.php:65]: Won't it be better to base it on Symfony version?

Comment (dannyvw) [reply src/Bundle/DependencyInjection/Compiler/PagerfantaBridgePass.php:65]: The `getDeprecation` method is added in the same PR, see https://github.com/symfony/symfony/pull/35778. So i think this should be good enough.

Comment (lchrusciel) [reply src/Bundle/DependencyInjection/Compiler/PagerfantaBridgePass.php:65]: I get it, I have the problem, that checking the number of arguments or Symfony version is a more explicit check for me. But perhaps, I'm just picky

**Comment by lchrusciel** [comment]
Thank you, Danny! :1st_place_medal:

**Pull request merged**